taxi_service/urls.py

```diff
@@ -22,4 +22,13 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    path("accounts/", include("django.contrib.auth.urls")),
+    path("__debug__/", include("debug_toolbar.urls")),
+] + static(
+    settings.STATIC_URL,
+    document_root=settings.STATIC_ROOT
+)
+
+if settings.DEBUG:
+    import debug_toolbar
+    urlpatterns += [path("__debug__/", include(debug_toolbar.urls))]
```

Comment:
If you are using debug toolbar and ur `DEBUG` value in settings is `False` - it will not work. So u don't need to check for `DEBUG` value here.

[Documentation](https://django-debug-toolbar.readthedocs.io/en/latest/installation.html)
```
urlpatterns = [
    # ... the rest of your URLconf goes here ...
] + debug_toolbar_urls()
```

Reply:
Yes, according to the documentation I should write it like this, but in that case my tests on git hub fail. I went to QnA and they told me to write it in path()